tests/v3_api/test_auth.py

```diff
@@ -104,7 +104,12 @@ def test_disable_and_enable_nestedgroup_for_openldap_set_access_control_restrict
 
 
 def test_ad_service_account_login():
+    delete_cluster_users()
+    auth_setup_data = setup["auth_setup_data"]
+    admin_user = auth_setup_data["admin_user"]
+    admin_token = login(admin_user, PASSWORD)
     if AUTH_PROVIDER == "activeDirectory":
+        enable_ad(admin_user, admin_token)
```

Comment:
@carolynhu does enable_ad logic already test the service account credentials? Is the login flow code different that enabling AD?